docker.d/configs/scriptworker.yaml

```diff
@@ -0,0 +1,56 @@
+provisioner_id: 'scriptworker-prov-v1'
+worker_group: 'treescriptworker-v1'
+worker_type:
+  $if: 'COT_PRODUCT == "thunderbird"'
+  then:
+    $if: 'ENV == "prod"'
+    then: 'treescript-dev'
+    else: 'treescript-v1'
+  else:
+    $if: 'ENV == "prod"'
+    then: 'treescript-dev'
+    else: 'treescript-v1'
+worker_id:
+  $eval: 'WORKER_ID'  # generated by configloader
+
+credentials:
+    clientId:
+      $eval: 'TASKCLUSTER_CLIENT_ID'  # set by cloudops
+    accessToken:
+      $eval: 'TASKCLUSTER_ACCESS_TOKEN'  # set by cloudops
+
+artifact_upload_timeout: 1200
+task_max_timeout: 3600
+task_script:
+  - 'app/bin/treescript'
+  - '/app/configs/worker_config.json'
+verbose: true
+
+verify_chain_of_trust:
+  $if: 'COT_PRODUCT == "thunderbird"'
```

Comment:
I'm not sure if we verify CoT for Thundebird in all env. I'd ask someone who knows.

Reply:
i copied this from https://github.com/mozilla-releng/build-puppet/blob/master/modules/tree_scriptworker/manifests/settings.pp#L50

Reply:
I think this works. For most scriptworkers, we verify CoT for both dev/stage and prod, because we download our `upstreamArtifacts` during CoT verification. However, treescript doesn't require any `upstreamArtifacts` currently, so we can skip CoT verification for dev/stage without breaking anything.

The question is whether we want to disable dev/stage treescript CoT verification for easier testing on Try, or if we want to enable it for consistency and making sure we catch graph/CoT errors before we deploy to production. I don't have a strong opinion here at the moment.

Reply:
I suggest we move forward how it was already configured in puppet and then we can revisit this at later point. I'd like to avoid doing too many changes at once.

@escapewindow @rail would that be ok?